src/dotnet/CommonLocalizableStrings.cs

```diff
@@ -5,183 +5,65 @@ namespace Microsoft.DotNet.Tools
 {
     internal class CommonLocalizableStrings
     {
-        public const string UnsupportedProjectType = "Unsupported project type. Please check with your sdk provider.";
         public const string ProjectAlreadyHasAreference = "Project already has a reference to `{0}`.";
         public const string ProjectReferenceCouldNotBeFound = "Project reference `{0}` could not be found.";
         public const string ProjectReferenceRemoved = "Project reference `{0}` removed.";
 
         // Project related
-        public const string Project = "Project";
```

Comment:
Which ones specifically? Should we move them into the `LocalizableStrings` section for a specific command?